doc/api/net.md

```diff
@@ -515,56 +544,74 @@ changes:
     description: The `hints` option is supported now.
 -->
 
-Opens the connection for a given socket.
-
-For TCP sockets, `options` argument should be an object which specifies:
-
-  - `port`: Port the client should connect to (Required).
-
-  - `host`: Host the client should connect to. Defaults to `'localhost'`.
+* `options` {Object}
+* `connectListener` {Function} Common parameters of [`socket.connect()`][]
+  methods. Will be added as a listener for the [`'connect'`][] event once.
+* Returns: {net.Socket} The socket itself.
 
-  - `localAddress`: Local interface to bind to for network connections.
+Initiate the connection on a given socket.
 
-  - `localPort`: Local port to bind to for network connections.
+Note: this method is similar to a [connect(2)][] call, and before that,
```

Comment:
btw, I don't have really strong feelings against pointing to the man pages, but if you do, I think you should list the system calls in the order they are called, rather than "connect, before that bind, before that getaddrinfo()..." (backwards).